omero/institution/getting-started.html

```diff
@@ -9,8 +9,108 @@
+        </div>
+        <hr>
+        <div class="row">
+            <div class="medium">
+                <p>A complete platform for managing your images, <a href="{{ site.baseurl }}/omero/" target="_blank">OMERO</a> has many <a href="{{ site.baseurl }}/omero/features" target="_blank">features</a> and <a href="{{ site.baseurl }}/omero/developers/" target="_blank">expansion points</a> to <a href="{{ site.baseurl }}/omero/features/analyze/" target="_blank"> integrate with your existing tools</a>. OMERO is <a href="{{ site.baseurl }}/omero/scientists/" target="_blank"> designed with scientists in mind</a> and is tried and tested in hundreds of institutions worldwide. In this section, you might get an idea what are the benefits of OMERO for your institution. 
+                </p>
+                <p>The software is for free, but do not miss the <a href="#what-is-needed">What is needed to host OMERO?</a> section below. Also [LINK TO NOTE ABOUT SUPPORT LEVELS].</p>
+            </div>
+            <div>
+                <h5>Selected OMERO features</h5>
+                <ul class="dev-ops-links">
+                    <li><a href="{{ site.baseurl }}/omero/features/view/" target="_blank">Access and view your images remotely via internet</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/features/share/" target="_blank">Cooperate on images with your colleagues</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/figure/" target="_blank">Create figures for scientific papers or colleagues</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/features/publish/" target="_blank">Publish your original image data</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/features/analyze/" target="_blank">Analyze the images in OMERO via API, using established third party image analysis software.</a></li>
+                </ul>
+                <h5>Read and watch about OMERO to get a better idea.</h5>
+                <ul class="dev-ops-links">
+                    <li><a href="{{ site.baseurl }}/omero/institution/" target="_blank">Further read on advantages of OMERO for your institution</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/features/index.html" target="_blank">List of OMERO features</a></li>
+                    <li><a href="https://omero-guides.readthedocs.io/en/latest/index.html" target="_blank">OMERO guides</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/index.html" target="_blank">OMERO documentation</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/apps/" target="_blank">List of OMERO Apps</a></li>
+                    <li><a href="https://www.youtube.com/channel/UCyySB9ZzNi8aBGYqcxSrauQ/videos" target="_blank">Watch videos on openmicroscopy YouTube channel</a></li>
+                    <li><a href="{{ site.baseurl }}/training/" target="_blank">See how to get a training on OMERO or organize one yourself</a></li>
```

Comment:
"get a training *session*"

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/462/commits/b34402411fec182b880b40934783bd667f93945f